### PR TITLE
🙄 Handle member roles

### PR DESCRIPTION
**Pull request by DevNikita16**
See #578

### CODE REVIEW

File: packages/ui/src/app/pages/Members/Members.tsx [hunk 1]
```diff
@@ -20,9 +19,9 @@ const sortReducer = (order: MemberListOrder, sortBy: MemberListSortKey): MemberL
   sortBy: sortBy,
   isDescending: sortBy === order.sortBy && !order.isDescending,
 })
-
-const Roles = Object.fromEntries(MemberRolesList.map(({ abbreviation }) => [abbreviation, abbreviation]))
-
+//
+// const Roles = Object.fromEntries(MemberRolesList.map(({ abbreviation }) => [abbreviation, abbreviation]))
+const Roles = {}
```
Comment (jodator) [inline packages/ui/src/app/pages/Members/Members.tsx:24]: Hey. Please remove empty comments. ;)

File: packages/ui/src/memberships/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+export function memberRoleTitle(role: MemberRole) {
+  if (role.groupName === 'Member Role') {
+    return role.groupName
+  }
+
+  const position = role.isLeader ? 'Leader' : 'Worker'
+  const group = role.groupName
+
+  return `${group} ${position}`
+}
```
Comment (jodator) [inline packages/ui/src/memberships/helpers.ts:20]: Overall the roles are working pretty nice now :)

I would like to ask you for two things:

1.  Add tests (Let's have it defined how this work with tests)
2.  The role should be formatted as "Forum Worker". However, we have "forum" in the group name so we should have the role name format independent of the underlying data.

**Review by jodator — CHANGES_REQUESTED** [review]
Overall the roles are working pretty nice now :)

Please see comments in the code.

File: packages/ui/src/memberships/helpers.ts (hunk 2 above)
Comment (DevNikita16) [reply packages/ui/src/memberships/helpers.ts:20]: @jodator, Just tests for this 2 functions?

Comment (jodator) [reply packages/ui/src/memberships/helpers.ts:20]: Yes, for these 2 functions.

**Comment by jodator** [comment]
@DevNikita16 please ping @p-sad when you update your PR.

@p-sad Please review the changes only (proper test names, etc). If all good just merge.

File: packages/ui/src/app/pages/Members/Members.tsx (hunk 1 above)
Comment (DevNikita16) [reply packages/ui/src/app/pages/Members/Members.tsx]: ✅ Added roles filter depending on members roles

File: packages/ui/src/memberships/helpers.ts (hunk 2 above)
Comment (DevNikita16) [reply packages/ui/src/memberships/helpers.ts]: ✅ Tests
✅ Capitalization

**Comment by DevNikita16** [comment]
@p-sad Ready for review 🥸

**Review by p-sad — APPROVED** [review]

**Pull request merged**